apps/concurrent/modules/user_input_parsing.pl

```diff
@@ -52,11 +52,12 @@
     get_action_and_val(Parts, Action, Value).
 
 add_to_assoc_list("impossible", Value, Assoc, New_Assoc) :-
-    member("impossible", Assoc)
-    ->  get_assoc("impossible", Assoc, Value_Sf),
-    	append(Value_Sf, [Value], New_Value),
+    (get_assoc("impossible", Assoc, Value_Sf)
```

Comment:
this was a bug: If there were multiple statements of type `impossible` for the same action, e.g. `impossible SHOOT at 5`, `impossible SHOOT at 7`, this was parsed to just the latter one.

Now fixed.